src/pages/admin/reports/components/amount-edit-row/AmountEditRow.module.scss

```
@@ -0,0 +1,23 @@
+@use '@/assets/sass/variables/colors' as c;
+@use '@/assets/sass/mixins/typography.mixins' as type;
+
+.td {
+    @include type.subtitle-2-medium;
+    box-sizing: border-box;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)